microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -20,15 +20,30 @@ These approaches are described below.
 
 
 <!-- ====================================================================== -->
-## Navigate to a file URL
+## Selecting an approach
+
+The approaches for loading local content into a WebView2 control support the following scenarios:
+
+| Scenario | Navigating to a file URL | Navigating to an HTML string | Virtual host name mapping | `WebResourceRequested` |
```

Comment:
```suggestion
| Scenario | By navigating to a file URL | By navigating to an HTML string | By using virtual host name mapping | By using `WebResourceRequested` |
```
to differentiate the "approaches" from the "scenarios" supported by each approach